controls/def.cf

```diff
@@ -162,6 +162,15 @@ bundle common def
                     trigger exec_command if any of them are defined.";
 
 
+      # Users allowed on sockets
+      "control_executor_runagent_socket_allow_users"
+        slist => { "cfapache" },
+        unless => isvariable( $(this.promiser) );
+
+      "control_executor_runagent_socket_allow_users"
+        slist => { @(def.control_executor_runagent_socket_allow_users) },
+        if => isvariable( $(this.promiser) );
```

Comment:
Shouldn't this be `if => isvariable("def.control_executor_runagent_socket_allow_users")`?

Reply:
Well, that's what this.promiser expands to (sans leading `def.`), but it should work with this.promiser. I will see if there is any difference when I test the build.

Reply:
It's still unclear to me how this works. The first promise defines a variable unless it is defined, the latter re-defines a variable if it is already defined?

Reply:
the second one was not necessary.